examples/operator/101_initial_cluster.yaml

```diff
@@ -27,25 +29,58 @@ spec:
       resources:
         requests:
           cpu: 100m
-          memory: 128Mi
+          memory: 256Mi
         limits:
-          memory: 512Mi
+          memory: 256Mi
   vitessDashboard:
     cells:
     - zone1
     extraFlags:
       security_policy: read-only
     replicas: 1
     resources:
+      limits:
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+  vtadmin:
+    rbac:
+      name: example-cluster-config
+      key: rbac.yaml
+    cells:
+      - zone1
+    apiAddresses:
+      - http://localhost:14001
+    replicas: 1
+    readOnly: false
+    apiResources:
+      limits:
+        memory: 128Mi
       requests:
         cpu: 100m
-        memory: 64Mi
+        memory: 128Mi
```

Comment:
Same thing here, maybe it would be good to ask @ajm188 or @notfelineit what they think the default resources allocation and the limit should be.

Reply:
I see that the old UI used to have a base request lower than the upper-bond limit

Reply:
I don't really have any expertise in this matter, I'll defer to them to decide.

Reply:
@notfelineit would be able to answer better than me (I have no idea!)

Reply:
The web resources look ok - I'm not sure about the api resources!

Reply:
If the example is running fine for Manan locally, this should be fine. Anyone running into issues on their environment can easily edit the provided yaml.